.github/pull_request_template.md

```diff
@@ -19,6 +19,16 @@ Bad example:
 change ingress
 -->
 
+### Risk Assessment
+
+<!-- Describe the risk of this change and how it will be tested, deployed, and verified. -->
+
+- [ ] Large/complex change?
+- [ ] Big splash zone?
+- [ ] Low margin for error?
```

Comment:
I am wondering if there is a different phrase we might use in place of "Low margin for error?".  At first blush I interpreted this item as a positive thing, i.e. "This change carries a low risk of causing errors", which was weird in the context of the other items in the list.

I was only able to come up with these options, the intent is kind of tough to phrase.  Hopefully they aren't too word-y:
- "High stakes if errors occur?"
- "Impacts data/systems which have a low tolerance for error?"
- "Impacts business-critical data/systems?"

Of course, I'm open to this just being a "me" problem.  The included reading in Guru clarified this checklist item well and I appreciate the included link in the description in case I forget.  Thanks for organizing this summary view for us, I think it'll be nice to have the risk assessment checks front-and-center when opening a PR.

Reply:
I doubt it's just a "you" problem now that you point it out.

Reply:
I think I'd vote for "High stakes if errors occur?" now.